### ISSUE TITLE
IasMonitor error in the logs

### ISSUE THREAD

**Issue by acaproni** [IntegratedAlarmSystem-Group/ias#157]
Sebastian found this error in the log of the IasMonitor that, according to his investigation, is also not publishing alarms

```ias-build-monitor              | 19:53:04.516 |ERROR| [KafkaStringsConsumer.java:443] [org.eso.ias.kafkautils.KafkaStringsConsumer-Thread] Consumer [MonitorID] got an exception got processing events: records lost!
ias-build-monitor              | com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "fullRunningId" (class org.eso.ias.heartbeat.serializer.HeartbeatMessagePojo), not marked as ignorable (4 known properties: "hbStringrepresentation", "state", "timestamp", "props"])
ias-build-monitor              |  at [Source: (String)"{"timestamp":"2019-03-07T19:53:04.531","fullRunningId":"(array:MONITORED_SOFTWARE_SYSTEM)@(UtilityModules:PLUGIN)","state":"RUNNING"}"; line: 1, column: 57] (through reference chain: org.eso.ias.heartbeat.serializer.HeartbeatMessagePojo["fullRunningId"])
ias-build-monitor              | 	at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:60)
ias-build-monitor              | 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:822)
ias-build-monitor              | 	at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:1152)
ias-build-monitor              | 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1582)
ias-build-monitor              | 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1560)
ias-build-monitor              | 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:294)
ias-build-monitor              | 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
ias-build-monitor              | 	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4001)
ias-build-monitor              | 	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2992)
ias-build-monitor              | 	at org.eso.ias.heartbeat.serializer.HbJsonSerializer.deserializeFromString(HbJsonSerializer.scala:51)
ias-build-monitor              | 	at org.eso.ias.heartbeat.consumer.HbKafkaConsumer.$anonfun$stringsReceived$1(HbKafkaConsumer.scala:145)
ias-build-monitor              | 	at java.util.ArrayList.forEach(ArrayList.java:1257)
ias-build-monitor              | 	at org.eso.ias.heartbeat.consumer.HbKafkaConsumer.stringsReceived(HbKafkaConsumer.scala:144)
ias-build-monitor              | 	at org.eso.ias.kafkautils.KafkaStringsConsumer.notifyListener(KafkaStringsConsumer.java:441)
ias-build-monitor              | 	at org.eso.ias.kafkautils.KafkaStringsConsumer.run(KafkaStringsConsumer.java:416)
ias-build-monitor              | 	at java.lang.Thread.run(Thread.java:748)
```

**Comment by acaproni** [comment]
From the logs, it looks like a plugin is sending the HB with the old format i.e. with the `fullRuningId` @see #145 .
This error does not explain the missing alarms from the IasMonitor

**Comment by acaproni** [comment]
For the error of the monitor the new version of the IAS is running but the the plugin are still from previous version.
About the missing alarms it is not clear from where they are missing. Have to check if they are on the `iasio.json`

**Comment by acaproni** [comment]
I double checked with Sebastian and we found the reason of the errors that was not due to errors in the code.

Testing we found that the monitoring tool takes the URL of kafka brokers from the CDB ignoring the java property. It has been fixed in the ` hotfix/9.0.1` branch.

I added the IASMON alarms to iasios.json.

We double checked and effectively th eold version of the plugins was in use and
